doc/contributing/maintaining/maintaining-dependencies.md

```diff
@@ -268,77 +281,72 @@ the main openssl/openssl releases with the addition of APIs to support
 the QUIC protocol.
 See [maintaining-openssl][] for more informations.
 
-### postject
+### postject 1.0.0-alpha.5
```

Comment:
Postject has recently been upgraded to 1.0.0-alpha.6, so this would need an update